COMMIT_EDITMSG

```diff
@@ -0,0 +1,3 @@
+Go to newly created distribution on save
```

Comment:
Do need to keep this file?

Reply:
Whoops. Womp womp. I think that was the result of an over-eager `git add`